x-pack/plugin/esql/qa/testFixtures/src/main/resources/lookup-join.csv-spec

```diff
@@ -325,3 +325,20 @@ count:long | type:keyword
 3          | Success
 1          | Disconnected
 ;
+
+sortEvalBeforeLookup
+required_capability: join_lookup_v4
```

Comment:
I think you need a new capability here

Reply:
It's just an optimisation, nothing changes in the output.

Reply:
Nah, that makes CI fail. Because the optimization is actually required to make this query pass at all. Otherwise, there are `OrderExec`s that are not turned into `TopNExec`s. And we can't plan `OrderExec`s, yet, as these can be unbounded.